Integrations/integration-SCADAfence_CNM.yml

```diff
@@ -317,6 +389,21 @@ script:
     elif demisto.command() == 'scadafence-getAssetConnections':
         params = map_optional_params(['ipAddress', 'macAddress', 'hostName'], ['ip', 'mac', 'host'])
         data = get_asset_map(params)
+        result = get_endpoint_data(data)
+        md = tableToMarkdown('Asset connections: ', result)
+        demisto.results({
+            'Type': entryTypes['note'],
+            'Contents': result,
+            'ContentsFormat': formats['json'],
+            'ReadableContentsFormat': formats['markdown'],
+            'HumanReadable': md,
+            'EntryContext': {
+                'SCADAfence.Asset.Conn(val.ip==obj.ip)':result
+            }
+        })
+
+    elif demisto.command() == 'scadafence-getAllConnections':
+        data = get_assets_map()
         md = tableToMarkdown('Asset connections: ', data)
```

Comment:
Shouldn't the title be something like `SCADAfence CNM connections` ? Not sure what `Asset` is that

Reply:
CNM is not connected to anything (just sniffing on a TAP or a SPAN port). Here we do get all assets interconnections in the traffic that the CNM sniffed at.